hub/api/tests/test_api.py

```diff
@@ -794,3 +794,13 @@ def test_tobytes(memory_ds, compressed_image_paths, audio_paths):
     for i in range(3):
         assert ds.image[i].tobytes() == image_bytes
         assert ds.audio[i].tobytes() == audio_bytes
+
+
+def test_empty_extend(memory_ds):
```

Comment:
Please add a doc line like "test extend empty tensor" or rename the test to something like `test_extend_empty_tensor`

Reply:
We are not testing extending an empty tensor, but rather extending a tensor with an empty list/array.